### ISSUE TITLE
Hello, may I ask what does the last / \  of this line in the losses.py mean?

### ISSUE THREAD

**Issue by yike5022** [lofrienger/S2ME#4]
y1 = -1*torch.sum(p*torch.log(p+1e-6), dim=1) / \
        torch.tensor(np.log(C)).cuda()
